Block/Adminhtml/Container.php

```diff
@@ -14,10 +14,12 @@
 use Magento\Backend\Block\Template\Context;
 use Magento\Framework\App\Request\Http;
 use Magento\Framework\Event\ManagerInterface;
+use Magento\Store\Model\StoreManagerInterface;
```

Comment:
Can we please keep these alphabetized? (Obviously Magento's style guide has no rule for this the same way the PHP SDK does, but I still think we should try to do it regardless.)

Reply:
... they are alphabetized F comes before S